audio/espeak-ng/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                espeak-ng
+github.setup        espeak-ng espeak-ng 1.50
+version             1.50
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         eSpeak NG is a multi-lingual software speech synthesizer
+
+long_description    eSpeak NG is a multi-lingual software speech synthesizer
+
+homepage            https://github.com/espeak-ng/espeak-ng
+distname            ${version}
+checksums           rmd160  bd3dd4e17863bb1aa3a964517d888ed9997f0e3e \
+                    sha256  f0505366d5391f1291a9c5cc0816461719f46eaed84cfbf57fb81b0dc381de21 \
+                    size    13665736
+
+depends_lib         port:pcaudiolib \
+                    port:sonic
+depends_build       port:rb26-ronn-ng \
+                    port:rb-kramdown \
+                    port:automake \
+                    port:autoconf \
+                    port:libtool \
+                    port:pkgconfig
+
+configure.cmd       ./autogen.sh && RONN=${prefix}/bin/ronn-2.6 ./configure --with-extdict-ru --with-extdict-zh --with-extdict-zhy
```

Comment:
Args (`--with-extdict-ru` ...) should not be specified in configure.cmd. Specify them in `configure.args` instead.

Environment variables (`RONN=...`) should not be specified in configure.cmd. Specify them in `configure.env` instead.

*If you need to autoreconf* then `configure.cmd       ./autogen.sh && ./configure` is acceptable although it is preferable to use:
```
use_autoreconf yes
autoreconf.cmd ./autogen.sh
```

If you make this change, make sure you append the autoconf, automake and libtool build dependencies *after* this line. If you have them before this line, they will be removed.

However, *do you really need to autoreconf*? If you were only doing this because the repository and therefore the auto-generated tarball does not contain a configure script, then switching to the release tarball as suggested elsewhere solves that problem.

Reply:
I could not get the configure from the release tarball to work...